src/widgets/AButton.cpp

```diff
@@ -390,10 +385,8 @@ void AButton::OnPaint(wxPaintEvent & WXUNUSED(event))
 
    mImages[mAlternateIdx].mArr[buttonState].Draw(dc, GetClientRect());
 
-   if( this == wxWindow::FindFocus() )
-   {
+   if(HasFocus())
```

Comment:
This change makes a difference for windows that override `GetMainWindowOfCompositeControl`.  What are such windows?

Reply:
Combos and list controls I see, but no such are used in the sidebar.

Reply:
You don't add any overrides in later commits.

But this change is harmless.